chirp-impl/src/test/java/sample/chirper/chirp/impl/ChirpServiceTest.java

```diff
@@ -17,17 +19,36 @@
 import scala.concurrent.duration.FiniteDuration;
 
 import java.time.Instant;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
 
 import static com.lightbend.lagom.javadsl.testkit.ServiceTest.*;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class ChirpServiceTest {
 
     private static TestServer server;
 
+    // Creating a new persistent entity is slow in some runtime environment (e.g. free tier Travis) causing the
+    // tests to fail on timeout. First time we invoke a call that requires a persistent entity we will
+    // wait a bit longer.
+    private static int firstRequestOnEntityTimeout;
+    private static final int secondRequestOnEntityTimeout = 1;
```

Comment:
Maybe a better naming would be:
secondRequestOnEntityTimeout -> defaultTimeout
firstRequestOnEntityTimeout -> delayedTimeout

Reply:
or `defautTimeout` and `firstRequestOnEntityTimeout`